<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterModel</class>
 <widget class="QDialog" name="FilterModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>过滤数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="ToolbtnWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active/>
       <inactive/>
       <disabled/>
      </palette>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="settingTb">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/images/images/(02,18).png</normaloff>:/images/images/(02,18).png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clusterTb">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>聚类分析</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/images/images/(42,24).png</normaloff>:/images/images/(42,24).png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="statisticalTb">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>统计</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/images/images/(48,09).png</normaloff>:/images/images/(48,09).png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveTb">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/images/images/(11,17).png</normaloff>:/images/images/(11,17).png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="tableWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="setting_widget" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>215</width>
              <height>335</height>
             </rect>
            </property>
            <attribute name="label">
             <string>聚类设置</string>
            </attribute>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>0</y>
               <width>263</width>
               <height>23</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="maxPower_checkBox">
                <property name="text">
                 <string>最大力</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="relPower_checkBox">
                <property name="text">
                 <string>相对力</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="perage_checkBox">
                <property name="text">
                 <string>百分比</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="contribution_checkBox">
                <property name="text">
                 <string>贡献率</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>40</y>
               <width>139</width>
               <height>32</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>聚类数</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="selClusterNum">
                <item>
                 <property name="text">
                  <string>请选择</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>215</width>
              <height>335</height>
             </rect>
            </property>
            <attribute name="label">
             <string>sssss</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTableView" name="resView"/>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="resLayout">
          <item>
           <widget class="QTableView" name="cluster_view"/>
          </item>
          <item>
           <widget class="QTableView" name="statistical_view"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterModel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterModel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
